compiler/arm/codegen/ARMDebug.cpp

```diff
@@ -1,19 +1,22 @@
 /*******************************************************************************
+ * Copyright (c) 2000, 2016 IBM Corp. and others
```

Comment:
Hm, technically this file is being changed in 2017 to update its copyright header....shouldn't we just unilaterally update the second date to 2017 in all cases?

Reply:
It was my understanding that the date should change only if something other than the copyright itself is different.

Reply:
I tend to agree with @keithc-ca that we should not update the dates because we are changing the copyright header. @mstoodle ?

Reply:
@mstoodle If you're ok with the dates, can you approve this so its obviously ready to merge once the PR builds pass?

Reply:
technically, the copyright statement applies to the copyright statement too :) but I think it's defensible not to change the date.